frontend/model/notifications/mainPeriodicNotificationEntries.js

```diff
@@ -0,0 +1,225 @@
+import { PROPOSAL_GENERIC, STATUS_OPEN } from '@model/contracts/shared/constants.js'
+import { DAYS_MILLIS, comparePeriodStamps, dateToPeriodStamp } from '@model/contracts/shared/time.js'
+import sbp from '@sbp/sbp'
+import { PERIODIC_NOTIFICATION_TYPE } from './periodicNotifications.js'
+import { extractProposalData } from './utils.js'
+
+// util functions
+const myNotificationHas = (checkFunc, groupId = '') => {
+  const myNotifications = groupId
+    ? sbp('state/vuex/getters').notificationsByGroup(groupId)
+    : sbp('state/vuex/getters').currentNotifications
+
+  return myNotifications.some(item => checkFunc(item))
+}
+
+const periodicNotificationEntries: {
+  type: string;
+  notificationData: {
+    stateKey: string;
+    emitCondition: (arg: { rootState: Object, rootGetters: Object }) => boolean;
+    emit: (arg: { rootState: Object, rootGetters: Object }) => void | Promise<void>;
+    shouldClearStateKey: (arg: { rootState: Object, rootGetters: Object }) => boolean;
+  }
+}[] = [
+  {
+    type: PERIODIC_NOTIFICATION_TYPE.MIN1,
+    notificationData: {
+      stateKey: 'nearDistributionEnd',
```

Comment:
Think this can be modified to `MIN15`, it's not something that needs to be checked every minute.

Reply:
Agreed